environment.yml

```diff
@@ -15,6 +15,8 @@ dependencies:
   - ncurses=6.1=hfc679d8_1
   - r-base
   - r-rcpp
+  - r-xml
+  - libgfortran-ng # needed by r packages
```

Comment:
Is libgfortran-ng really needed? i think this is already installed if you have the fortran compiler installed.